sw/device/silicon_creator/lib/drivers/spi_device.c

```diff
@@ -520,11 +520,12 @@ void spi_device_init(void) {
   // generation fields of the device ID (3 and 4 bits, respectively).
   lifecycle_hw_rev_t hw_rev;
   lifecycle_hw_rev_get(&hw_rev);
+  // TODO: hw_rev mapping may have to be updated.
```

Comment:
@alphan @dmcardle @moidx @timothytrippel @cfrantz 

This may need an update in order to adjust the bitmapping.

Reply:
Does lifecycle_hw_rev_get return the correct values?

Reply:
Just checked the diff, LGTM. We just need to update the expected value in that test.

Reply:
```
diff --git a/sw/host/tests/rom/e2e_bootstrap_entry/src/main.rs b/sw/host/tests/rom/e2e_bootstrap_entry/src/main.rs
index 0e3183d38..d54b4b2f7 100644
--- a/sw/host/tests/rom/e2e_bootstrap_entry/src/main.rs
+++ b/sw/host/tests/rom/e2e_bootstrap_entry/src/main.rs
@@ -49,7 +49,7 @@ struct Opts {
 
     #[structopt(
         long, parse(try_from_str=u8::from_str),
-        default_value = "0x08",
+        default_value = "0x19",
         help = "JEDEC manufacturer product ID",
     )]
     jedec_product: u8,
```

Reply:
Cool, thanks!